packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1875,21 +1886,21 @@ abstract class RenderBox extends RenderObject {
   ///
   /// When implementing a [RenderBox] subclass, to override the baseline
   /// computation, override [computeDistanceToActualBaseline].
-  double getDistanceToBaseline(TextBaseline/*!*/ baseline, { bool onlyReal = false }) {
+  double? getDistanceToBaseline(TextBaseline baseline, { bool onlyReal = false }) {
     assert(!_debugDoingBaseline, 'Please see the documentation for computeDistanceToActualBaseline for the required calling conventions of this method.');
-    assert(!debugNeedsLayout);
+    assert(!debugNeedsLayout!);
```

Comment:
we should make debugNeedsLayout non-nullable somehow, just have it throw in release mode (i assume it's nullable just because in release mode it returns null, i haven't checked)

Reply:
Changed to make debugNeedsLayout non-nullable